rpc/rpc-lib/src/rpcsvc.c

```diff
@@ -2546,9 +2547,9 @@ rpcsvc_reconfigure_options(rpcsvc_t *svc, dict_t *options)
     /* Reconfigure the volume specific rpc-auth.addr allow part */
     volentry = xlator->children;
     while (volentry) {
-        ret = gf_asprintf(&srchkey, "rpc-auth.addr.%s.allow",
-                          volentry->xlator->name);
-        if (ret == -1) {
+        keylen = gf_asprintf(&srchkey, "rpc-auth.addr.%s.allow",
+                             volentry->xlator->name);
+        if (keylen == -1) {
```

Comment:
nit: I think these days we prefer < 0 or <= 0

```suggestion
        if (keylen <= 0) {
```